migrationTool/ams/AssetMigrationTracker.cs

```diff
@@ -129,9 +129,9 @@ public async Task UpdateMigrationStatus(BlobContainerClient container, AssetMigr
                 metadata.Add(ManifestNameKey, result.ManifestName);
             }
 
-            if (result.OutputPath != null)
+            if (result.OutputPath != null && !string.IsNullOrEmpty(result.OutputPath.AbsoluteUri))
```

Comment:
The string.IsNullOrEmpty() check is not needed since the OutputPath is a URL object which is not null